/**:
  ros__parameters:
    fuse_unknown_only: true
    min_cluster_size: 2
    cluster_2d_tolerance: 0.5
